From 01254efa6f69078000da7e87e43c9f245df0cfc0 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Sat, 10 Nov 2018 18:44:47 +0100
Subject: build: add missing l_genl_family_set_unicast_handler

Add l_genl_family_set_unicast_handler (and remove legacy
l_genl_set_unicast_handler).

Fixes iwd compile failure:

  src/netdev.o: In function `netdev_set_nl80211':
  netdev.c:(.text+0x6f04): undefined reference to `l_genl_family_set_unicast_handler'
---
 ell/ell.sym | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ell/ell.sym b/ell/ell.sym
index 614bc93..c21fb21 100644
--- ell/ell.sym
+++ ell/ell.sym
@@ -239,7 +239,6 @@ global:
 	l_genl_unref;
 	l_genl_set_debug;
 	l_genl_set_close_on_unref;
-	l_genl_set_unicast_handler;
 	l_genl_msg_new;
 	l_genl_msg_new_sized;
 	l_genl_msg_ref;
@@ -257,6 +256,7 @@ global:
 	l_genl_family_new;
 	l_genl_family_ref;
 	l_genl_family_unref;
+	l_genl_family_set_unicast_handler;
 	l_genl_family_set_watches;
 	l_genl_family_get_version;
 	l_genl_family_get_genl;
-- 
cgit 1.2-0.3.lf.el7


